lib/phoenix_live_view/html_algebra.ex

```diff
@@ -12,9 +12,18 @@ defmodule Phoenix.LiveView.HTMLAlgebra do
   #
   def build(tree, opts) when is_list(tree) do
     {migrate, opts} = Keyword.pop(opts, :migrate_eex_to_curly_interpolation, true)
+    {attr_formatters, opts} = Keyword.pop(opts, :attribute_formatters, %{})
+
+    attr_formatters =
+      Map.new(attr_formatters, fn {attr, formatter} -> {to_string(attr), formatter} end)
```

Comment:
We should do this normalization in the HTMLFormatter and, at that moment, validate if the formatter module is loaded and, if not, log and discard it. That's so we can still format even with dependencies partially loaded.